COMAN:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  
  
  # Position Controllers ---------------------------------------
  #TORSO
  WaistSag_position_controller:
    type: effort_controllers/JointPositionController
    joint: WaistSag
    pid: {p: 1000.0, i: 0.1, d: 3.0}

  WaistLat_position_controller:
    type: effort_controllers/JointPositionController
    joint: WaistLat
    pid: {p: 1000.0, i: 0.1, d: 3.0}

  WaistYaw_position_controller:
    type: effort_controllers/JointPositionController
    joint: WaistYaw
    pid: {p: 1000.0, i: 0.1, d: 3.0}

  #RIGHT LEG
  RHipSag_position_controller:
    type: effort_controllers/JointPositionController
    joint: RHipSag
    pid: {p: 1000.0, i: 0.1, d: 3.0}

  RHipLat_position_controller:
    type: effort_controllers/JointPositionController
    joint: RHipLat
    pid: {p: 1000.0, i: 0.1, d: 3.0}

  RHipYaw_position_controller:
    type: effort_controllers/JointPositionController
    joint: RHipYaw
    pid: {p: 1000.0, i: 0.1, d: 3.0}

  RKneeSag_position_controller:
    type: effort_controllers/JointPositionController
    joint: RKneeSag
    pid: {p: 1000.0, i: 0.1, d: 3.0}

  RAnkLat_position_controller:
    type: effort_controllers/JointPositionController
    joint: RAnkLat
    pid: {p: 1000.0, i: 0.1, d: 3.0}

  RAnkSag_position_controller:
    type: effort_controllers/JointPositionController
    joint: RAnkSag
    pid: {p: 1000.0, i: 0.1, d: 3.0}

  #LEFT LEG
  LHipSag_position_controller:
    type: effort_controllers/JointPositionController
    joint: LHipSag
    pid: {p: 1000.0, i: 0.1, d: 3.0}

  LHipLat_position_controller:
    type: effort_controllers/JointPositionController
    joint: LHipLat
    pid: {p: 1000.0, i: 0.1, d: 3.0}

  LHipYaw_position_controller:
    type: effort_controllers/JointPositionController
    joint: LHipYaw
    pid: {p: 1000.0, i: 0.1, d: 3.0}

  LKneeSag_position_controller:
    type: effort_controllers/JointPositionController
    joint: LKneeSag
    pid: {p: 1000.0, i: 0.1, d: 3.0}

  LAnkLat_position_controller:
    type: effort_controllers/JointPositionController
    joint: LAnkLat
    pid: {p: 1000.0, i: 0.1, d: 3.0}

  LAnkSag_position_controller:
    type: effort_controllers/JointPositionController
    joint: LAnkSag
    pid: {p: 1000.0, i: 0.1, d: 3.0}

#RIGHT ARM
  RShSag_position_controller:
    type: effort_controllers/JointPositionController
    joint: RShSag
    pid: {p: 300.0, i: 0.1, d: 1.0}

  RShLat_position_controller:
    type: effort_controllers/JointPositionController
    joint: RShLat
    pid: {p: 300.0, i: 0.1, d: 1.0}

  RShYaw_position_controller:
    type: effort_controllers/JointPositionController
    joint: RShYaw
    pid: {p: 300.0, i: 0.1, d: 1.0}

  RElbj_position_controller:
    type: effort_controllers/JointPositionController
    joint: RElbj
    pid: {p: 300.0, i: 0.1, d: 0.1}

#LEFT ARM
  LShSag_position_controller:
    type: effort_controllers/JointPositionController
    joint: LShSag
    pid: {p: 300.0, i: 0.1, d: 1.0}

  LShLat_position_controller:
    type: effort_controllers/JointPositionController
    joint: LShLat
    pid: {p: 300.0, i: 0.1, d: 1.0}

  LShYaw_position_controller:
    type: effort_controllers/JointPositionController
    joint: LShYaw
    pid: {p: 300.0, i: 0.1, d: 1.0}

  LElbj_position_controller:
    type: effort_controllers/JointPositionController
    joint: LElbj
    pid: {p: 300.0, i: 0.1, d: 0.1}
